plugin/src/sbt-test/scalafmt-sbt/sbt/build.sbt

```diff
@@ -5,6 +5,16 @@ ThisBuild / fork := true
 
 lazy val p123 = project
   .in(file("."))
+  .settings(
+    InputKey[Unit]("updateLastModified") := {
+      import sbt.internal.util.complete.DefaultParsers._
+      import scala.sys.process.Process
+      val f = spaceDelimited("").parsed.head
```

Comment:
what does this do?

Reply:
get inputTask first argument.

e.g.
first argument is `p13/src/main/scala/Test.scala` if test script is

```
> updateLastModified p13/src/main/scala/Test.scala
```

Reply:
does this method work with filenames containing spaces?

Reply:
works if add quotes.

e.g.

```
updateLastModified "p13/src/main/scala/file name with space.scala"
```